core/arch/arm/plat-vexpress/platform_config.h

```diff
@@ -110,22 +110,22 @@
  */
 
 #define DRAM0_BASE		0x80000000
-#define DRAM0_SIZE		0x80000000
+#define DRAM0_SIZE		0x7F000000
 
 #ifdef CFG_WITH_PAGER
 
 /* Emulated SRAM */
-#define TZSRAM_BASE		(0x06000000)
+#define TZSRAM_BASE		0xFF000000
 #define TZSRAM_SIZE		CFG_CORE_TZSRAM_EMUL_SIZE
 
 #define TZDRAM_BASE		(TZSRAM_BASE + CFG_TEE_RAM_VA_SIZE)
-#define TZDRAM_SIZE		(0x02000000 - CFG_TEE_RAM_VA_SIZE)
+#define TZDRAM_SIZE		(0x01000000 - CFG_TEE_RAM_VA_SIZE)
 
 #else /*CFG_WITH_PAGER*/
 
 /* Location of trusted dram on the base fvp */
```

Comment:
Should this comment also be updated so it aligns with the commit message, I.e, say that it is the secure region in DRAM configured by the TrustZone controller. Do you know if there is a memory map? If there is, it would be nice to link that too in the commit message.